cyberark/conjur
# 382 - Conjur.org Main Navigation & Structure Changes

Closes #382

#### What does this pull request do?

This PR changes the navigation structure and order, and adds a second level of navigation per #382.

#### What background context can you provide?

We've made the primary nav more intuitive and added the ability for more complex navigation menus.

#### Where should the reviewer start?

The nav structure is in _docs/_data/navigation.yml_, and the new tools menu is determined in _docs/_data/toc.yml_.  Other than that, a few files got moved (and indexes created) to support the new page hierarchy, and there are changes to the CSS and JS to support the second level menu.

#### How should this be manually tested?

Reviewers should pull down this branch and build the docs project per the instructions in https://github.com/cyberark/conjur/blob/master/docs/README.md - if the site files haven't regenerated properly, you can also `cd` into the _/docs/_ subfolder in the project and run `jekyll build --watch` to have it forcibly regenerate the static files.

Testing in the browser full-width and on a tablet and mobile phone would be helpful!

#### Screenshots (if appropriate)

**Desktop:**
![image](https://user-images.githubusercontent.com/4164335/30746382-aac6747e-9f77-11e7-9d51-adf132df6bfd.png)

**Tablet:**
![image](https://user-images.githubusercontent.com/4164335/30746435-cb6bf5fa-9f77-11e7-8c6f-3d67987d8227.png)

**Phone:**
![image](https://user-images.githubusercontent.com/4164335/30746498-ecc738f4-9f77-11e7-894c-5118a48527e6.png)

#### Link to build in Jenkins (if appropriate)

https://jenkins.conjur.net/job/cyberark--conjur/job/382-nav-updates/
